tablo/models.py

```diff
@@ -542,30 +581,30 @@ def get_quantile_breaks(self, field, break_count):
 
     def get_natural_breaks(self, field, break_count):
 
-        if not self._validate_fields([field]):
-            raise ValueError('Invalid field')
-
-        sql_statement = """SELECT service_table.{field} FROM
-                            (SELECT ROW_NUMBER() OVER (ORDER BY {field}) AS row_number,
```

Comment:
This change was also motivated by formatting for readability, but I also:

1. moved the one non-subquery table in the `FROM` clause so that it appears first in the table list
2. simplified the `row_number = 1 OR row_number = "total"` in the `WHERE` clause to a `row_number IN ...`

I tested this one by writing similar SQL to ensure that reordered `FROM` tables behave the same way, and that `IN` clauses that reference variables and constants are legal in PostgreSQL. However, I'm not sure when these blocks of code execute in terms of the CSV dataset life cycle, so I'm not sure if the queries themselves have been tested through the app.